pkgs/applications/snowmachine/default.nix

```
@@ -0,0 +1,24 @@
+{ buildPythonPackage, lib, click, colorama, fetchPypi, setuptools-git }:
+
+buildPythonPackage rec {
+  pname = "snowmachine";
+  version = "1.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1v385hhxy2a8vx5p0fhn0di8l4qfpb0a86j6nwsg0aw6ngb09qf1";
+  };
+
+  buildInputs = [ setuptools-git ];
+  propagatedBuildInputs = [ click colorama ];
+
+  doCheck = false;
+  pythonImportsCheck = [ "click" "colorama" ];
```

```suggestion
  pythonImportsCheck = [ "snowmachine" ];
```

It should test the package we are building not a dependency.